### PR TITLE
Remove unnecessary null check

### PR DESCRIPTION
**Pull request by nezihyigitbasi**
Checked value is a primitive.

### CODE REVIEW

File: presto-main/src/main/java/com/facebook/presto/execution/QueryStats.java [hunk 1]
```diff
@@ -209,7 +209,7 @@ public QueryStats(
         checkArgument(completedDrivers >= 0, "completedDrivers is negative");
         this.completedDrivers = completedDrivers;
 
-        this.cumulativeUserMemory = requireNonNull(cumulativeUserMemory, "cumulativeUserMemory is null");
+        this.cumulativeUserMemory = cumulativeUserMemory;
```
Comment (dain) [inline presto-main/src/main/java/com/facebook/presto/execution/QueryStats.java]: Do you want to check if it is non-negative

File: presto-main/src/main/java/com/facebook/presto/execution/StageStats.java [hunk 2]
```diff
@@ -178,7 +178,7 @@ public StageStats(
         checkArgument(completedDrivers >= 0, "completedDrivers is negative");
         this.completedDrivers = completedDrivers;
 
-        this.cumulativeUserMemory = requireNonNull(cumulativeUserMemory, "cumulativeUserMemory is null");
+        this.cumulativeUserMemory = cumulativeUserMemory;
```
Comment (dain) [inline presto-main/src/main/java/com/facebook/presto/execution/StageStats.java]: Do you want to check if it is non-negative

**Comment by findepi** [comment]
> Checked value is a primitive.

This just happens.. Would it be possible to flag such things automatically?

**Comment by nezihyigitbasi** [comment]
@findepi Sounds possible theoretically, but don't know whether we can do it with checkstyle.

**Comment by findepi** [comment]
Sounds more like findbugs ....

>

**Pull request merged**